ios/Classes/SwiftVibrationPlugin.swift

```diff
@@ -15,6 +15,8 @@ public class SwiftVibrationPlugin: NSObject, FlutterPlugin {
         switch (call.method) {
         case "hasVibrator":
             result(isDevice)
+	case "hasAmplitudeControl":
+            result(isDevice)
```

Comment:
Maybe it makes sense to use `nil` here or something? iOS has no support for it anyway.

Reply:
I do not have a method of testing iOS, so was simply following the existing convention. I did a bit of searching beforehand and found the AudioServicesPlaySystemSoundWithVibration() API, which includes support for both the vibration pattern and the intensities on iOS, so it would seem that the building blocks are certainly available. I did, however, not feel comfortable making any changes here on account of having no means to test.

I would guess that a nil return here would cause the platform channel method to blow up in the same way, which we avoided with a false return on Android - perhaps we can use the same approach for iOS until someone has the opportunity to try out the AudioServicesPlaySystemSoundWithVibration API?